src/components/header-card-action-buttons/header-card-action-buttons.styles.ts

```diff
@@ -0,0 +1,7 @@
+import { createUseStyles } from '../../styles/create-use-styles';
+
+export const useHeaderCardActionButtonsStyles = createUseStyles(() => ({
+  buttonContainer: {
```

Comment:
I think you should not use this for the `RECEIVE` and `BUY ` buttons.  
This does nothing to them.